services/web/server/src/simcore_service_webserver/notifications/_rabbitmq_exclusive_queue_consumers.py

```diff
@@ -151,6 +155,44 @@ async def _events_message_parser(app: web.Application, data: bytes) -> bool:
     return True
 
 
+async def _webserver_internal_events_message_parser(
+    app: web.Application, data: bytes
+) -> bool:
+    """
+    Handles internal webserver events that need to be propagated to other webserver replicas
+
+    Ex. Log unsubscription is triggered by user closing a project, which is a REST API call
+    that can reach any webserver replica. Then this event is propagated to all replicas
+    so that the one holding the websocket connection can unsubscribe from the logs queue.
+    """
+
+    rabbit_message = WebserverInternalEventRabbitMessage.model_validate_json(data)
+
+    if (
+        rabbit_message.action
+        == WebserverInternalEventRabbitMessageAction.UNSUBSCRIBE_FROM_PROJECT_LOGS_RABBIT_QUEUE
+    ):
+        _project_id = rabbit_message.data.get("project_id")
+
+        if _project_id:
+            _logger.debug(
+                "Received UNSUBSCRIBE_FROM_PROJECT_LOGS_RABBIT_QUEUE event for project %s",
+                _project_id,
+            )
+            await project_logs.unsubscribe(app, ProjectID(_project_id))
+        else:
+            _logger.exception(
+                "Missing project_id in UNSUBSCRIBE_FROM_LOGS_RABBIT_QUEUE event, this should never happen, investigate!"
+            )
+
+    else:
+        _logger.warning(
+            "Unknown webserver internal event message action %s", rabbit_message.action
+        )
+
+    return True
```

Comment:
what is the purpos of thie return? should it return False if the condition above is not satisfied?